src/common/src/array/stream_chunk.rs

```diff
@@ -276,13 +276,22 @@ impl StreamChunk {
 
 impl fmt::Debug for StreamChunk {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(
-            f,
-            "StreamChunk {{ cardinality = {}, capacity = {}, data = \n{} }}",
-            self.cardinality(),
-            self.capacity(),
-            self.to_pretty_string()
-        )
+        if f.alternate() {
+            write!(
+                f,
+                "StreamChunk {{ cardinality = {}, capacity = {}, data = \n{} }}",
+                self.cardinality(),
+                self.capacity(),
+                self.to_pretty_string()
+            )
+        } else {
+            write!(
+                f,
+                "StreamChunk {{ cardinality = {}, capacity = {}, data = ... }}",
```

Comment:
How about refactoring with `debug_struct`? This can be represented with an optional `finish_non_exhaustive`.

https://github.com/risingwavelabs/risingwave/blob/3d067c7e9e66688d7d16bb6faa27e6267047b5b0/src/stream/src/executor/exchange/input.rs#L53-L59

Reply:
However, it's not applicable to the alternated format, because the `self.to_pretty_string()` need to be printed directly without being quoted in `"`